testhelper/testhelper.go

```diff
@@ -204,7 +298,7 @@ func CreateIntegrationSLAMService(
 		return nil, err
 	}
 	test.That(t, sensorDeps, test.ShouldResemble, []string{cfg.Camera["name"]})
```

Comment:
[note] Why are we only looking for camera in sensor dependencies? Also why don't we pass in that sensorsDeps into `SetupStubDeps`?

Reply:
Since IMU is optional, it is set up in GetOptionalParameters, rather than being passed in to the Validate function. 
As to Q2, we never did... see https://github.com/viamrobotics/viam-cartographer/blob/9797b6aa4518543a647a0dfbb1be9f44f1c9105f/testhelper/testhelper.go#L207

Reply:
Spoke offline, adding optional IMU to implicit dependencies in Validate

Reply:
Done